src/pages/DeepExplorePage.tsx

```diff
@@ -0,0 +1,56 @@
+import { Country } from '../services/api-types';
+import { DateRange, SamplingStrategy } from '../services/api';
+import { DeepRoute, makeLayout, makeSwitch } from '../helpers/deep-page';
+import { Button } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import React from 'react';
+import { Route, useRouteMatch } from 'react-router';
+import { SequencingCoverageDeepExplore } from '../components/SequencingCoverageDeepExplore';
+import { SequencingIntensityEntrySetWithSelector } from '../helpers/sequencing-intensity-entry-set';
+
+interface Props {
+  country: Country;
+  dateRange: DateRange;
+  samplingStrategy: SamplingStrategy;
+  sequencingIntensityEntrySet: SequencingIntensityEntrySetWithSelector;
+}
+
+const routes: DeepRoute<Props>[] = [
+  {
+    key: 'sequencing-coverage',
+    title: 'Sequencing Coverage',
+    content: props => (
+      <SequencingCoverageDeepExplore
+        country={props.country}
+        dateRange={props.dateRange}
+        samplingStrategy={props.samplingStrategy}
+        sequencingIntensityEntrySet={props.sequencingIntensityEntrySet}
+      />
+    ),
+  },
+];
+
+export const DeepExplorePage = (props: Props) => {
+  const { path, url } = useRouteMatch();
+  const _makeLayout = (content: JSX.Element) =>
```

Comment:
why does this have an undersscore?

Reply:
Because it is importing a function called `makeLayout` from `'../helpers/deep-page'` (line 3)